base.js

```diff
@@ -13,6 +13,22 @@ module.exports = {
 		],
 		'no-console': ['error', { allow: ['error', 'info', 'warn'] }],
 		'no-tabs': 'off',
+		'no-await-in-loop': 'off',
+		'no-restricted-syntax': [
+			'error',
+			{
+				selector: 'ForInStatement',
+				message: 'for..in loops iterate over the entire prototype chain, which is virtually never what you want. Use Object.{keys,values,entries}, and iterate over the resulting array.',
+			},
+			{
+				selector: 'LabeledStatement',
+				message: 'Labels are a form of GOTO; using them makes code confusing and hard to maintain and understand.',
+			},
+			{
+				selector: 'WithStatement',
+				message: '`with` is disallowed in strict mode because it makes code impossible to predict and optimize.',
+			},
+		],
 		// Prettier conflicts
 		curly: 'off',
```

Comment:
Not related, but why does this not have `'` surrounding the word `curly`?

Reply:
Prettier removes them when property is just one word.